lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -481,29 +480,36 @@ def test_build_student_data_limit(self):
             student = self.create_student('student{}'.format(ctr))
             self.submit_student_answer(student.username, u'Problem1', ['Option 1'])
 
-        student_data = ProblemResponses._build_student_data(user_id=self.instructor.id,
-                                                            course_id=self.course.id,
-                                                            problem_location=str(self.course.location))
+        student_data = ProblemResponses._build_student_data(
+            user_id=self.instructor.id,
+            course_id=self.course.id,
+            problem_location=str(self.course.location),
+        )
 
         self.assertEquals(len(student_data), 4)
 
     def test_build_student_data(self):
         self.define_option_problem(u'Problem1')
         self.submit_student_answer(self.student.username, u'Problem1', ['Option 1'])
 
-        student_data = ProblemResponses._build_student_data(user_id=self.instructor.id,
-                                                            course_id=self.course.id,
-                                                            problem_location=str(self.course.location))
+        student_data = ProblemResponses._build_student_data(
+            user_id=self.instructor.id,
+            course_id=self.course.id,
+            problem_location=str(self.course.location),
+        )
         self.assertEquals(len(student_data), 1)
         self.assertDictContainsSubset({
             'username': 'student',
-            'location': 'test_course > Section > Subsection',
-            'block_id': 'Problem1',
+            'location': 'test_course > Section > Subsection > Problem1',
+            'block_key': 'i4x://edx/1.23x/problem/Problem1',
```

Comment:
`i4x:` is the old key format... do you have a choice here to use `xblock-v1:...`, or is that just what this test is already using?

Reply:
I got this by using pdb while running the test, so I just captured what was returned. I guess we could just remove the block_key comparison if the exact format doesn't matter, and could change.

Reply:
Ok, it's not a big deal. Fine to leave as is.